core/modules/map.rs

```diff
@@ -145,16 +133,18 @@ impl ModuleMap {
       for m in not_evaluated {
         msg.push_str(&format!("  - {}\n", m));
       }
-      panic!("{}", msg);
+      // panic!("{}", msg);
```

Comment:
Uncomment before landing, this is a useful debugging tool.

Reply:
Ack, missed this. Good catch